practice-two/src/style/components/button.css

```diff
@@ -0,0 +1,44 @@
+.btn {
+  font-size: var(--font-size-2xs);
+  font-style: var(--font-style-default);
+  font-weight: var(--font-weight-primary);
+  line-height: var(--line-height-normal);
+  letter-spacing: var(--letter-spacing-lg);
+  border-width: var(--border-btn-width-default);
+  border-style: var(--border-btn-style-default);
+  border-color: var(--color-border-secondary);
```

Comment:
You can use shorthand `border: var(--border-btn-width-default) var(--border-btn-style-default) var(--color-border-secondary)`